themes/isp_uv/layouts/seminars/list.html

```diff
@@ -1,71 +1,55 @@
 {{ define "main" }}
-<div class="content-container" style="font-size: 14px;">
+<div class="content-container" style="font-size: 14px; width: 100%;">
 
   <h1 style="margin-bottom: 20px;">Seminars</h1>
 
-  <div class="course-container" style="border-bottom: 0px solid #ccc;">
-    <div class="image-container">
-      <img src="/isp/images/talks2.jpg" height="100" width="260">
+  <div class="course-container" style="border-bottom: 0px solid #ccc; display: flex;">
+    <div class="image-container" style="flex: 0 0 260px;">
+      <img src="/images/talks2.webp" height="100" width="260" style="max-width: 100%;">
     </div>
-    <div class="text-container">
+    <div class="text-container" style="flex: 1; padding-left: 20px;">
       In the Image Processing Lab (IPL) environment, invited talks by senior researchers, postdocs and PhD students in
       meetings for informal discussion of their research are held regularly, organized by the ISP group. We call these
       talks FFT, that in this case stands for "First Friday Talk", even though they may happen at any point of the week
       ;-) Below you may find a (non-exhaustive) list of the talks done so far to get a flavour of our interests...
     </div>
   </div>
 
-  <table class="table table-hover table-condensed table-responsive" id="seminar-table">
+  <table class="table table-hover table-condensed table-responsive" id="seminar-table" style="width: 100%;">
     <thead>
       <tr>
-        <th style="background-color: #f2dede">Date</th>
-        <th style="background-color: #f2dede">Title</th>
-        <th style="background-color: #f2dede">Speaker</th>
-        <th style="background-color: #f2dede">Affiliation</th>
+        <th style="background-color: #f2dede; width: 15%;">Date</th>
+        <th style="background-color: #f2dede; width: 50%;">Title</th>
+        <th style="background-color: #f2dede; width: 20%;">Speaker</th>
+        <th style="background-color: #f2dede; width: 15%;">Affiliation</th>
       </tr>
     </thead>
-    <tbody id="seminar-list"></tbody>
+    <tbody>
+      {{ $currentYear := "" }}
+      {{ range .Site.RegularPages }}
+        {{ if eq .Section "seminars" }}
+          {{ $year := .Date.Format "2006" }}
+          {{ if ne $currentYear $year }}
+            {{ if ne $currentYear "" }}
```

Comment:
I think you could simplify this using the `GroupByDate` method check https://gohugo.io/methods/pages/groupbydate/